.buildkite/pipeline.yml

```diff
@@ -107,6 +107,26 @@ steps:
         artifact_paths:
           - "**/build/test-results/merged-test-results.xml"
 
+      - label: "🔬 Unit test FluxC"
+        command: ".buildkite/commands/run-unit-tests.sh fluxc"
+        plugins:
+          - $CI_TOOLKIT
+          - $TEST_COLLECTOR :
+              <<: *test_collector_common_params
+              api-token-env-name: "BUILDKITE_ANALYTICS_TOKEN_UNIT_TESTS_WORDPRESS"
```

Comment:
@mokagio might know more as I think he has more experience in setting up Buildkite Test Analytics, but iinm the way BTA works it uses the token to know which Test Analytics Suite to associate that run with (eg https://buildkite.com/organizations/automattic/analytics/suites/fluxc-android-unit-tests/suite_token)

So the use of different tokens here is not a matter of security or similar but more because we want those different types of tests to be reported in different "Test Suites" in Buildkite, so that the analytics for each don't all end up in the same stats bucket but instead have their own independent graphs and stats in Buildkite.

Reply:
> So the use of different tokens here is not a matter of security or similar but more because we want those different types of tests to be reported in different "Test Suites" in Buildkite, so that the analytics for each don't all end up in the same stats bucket but instead have their own independent graphs and stats in Buildkite.

Yeah, that was the reason. A bit more context in paaHJt-3wn-p2#comment-6208

Reply:
I see, thanks! I've added new test suites and added tokens to CI's `env`. Soon, I update this pipeline.